com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/BuildCloudDeployModelStep.java

```diff
@@ -100,6 +103,23 @@ protected StepPhase executeStep(ExecutionWrapper execution) throws SLException {
         }
     }
 
+    private void buildConfigurationEntriesToPublish(DelegateExecution context, DeploymentDescriptor deploymentDescriptor,
+        List<CloudApplicationExtended> apps) {
+        ProcessType processType = processTypeParser.getProcessType(context);
+
+        if (processType.equals(ProcessType.BLUE_GREEN_DEPLOY)) {
+            StepsUtil.setConfigurationEntriesToPublish(context, Collections.emptyMap());
+            StepsUtil.setSkipUpdateConfigurationEntries(context, true);

```

Comment:
Could we remove this skipUpdateConfigurationEntries variable and calculate whether it should be true or false based on the configurationEntriesToPublish?

Reply:
Actually, configurationEntriesToPublish are build depending if configuration entries will be updated or skiped.
In parent process in PrepareAppsDeploymentStep StepsUtil.setSkipUpdateConfigurationEntries is invoked.
In sub-process in RebuildApplicationDeployModelStep configurationEntries are calculated based on "skip" flag.